theme-kit/lib/locales.js

```diff
@@ -0,0 +1,112 @@
+/**
+ * This list is pulled from the main Prezly application. The underscores are replaced with dashes.
+ */
+module.exports = [
```

Comment:
This is not used yet. Probably, I will remove it, as the router does not need a list of hard-coded locales anymore.